infot.mod

```diff
@@ -411,6 +417,7 @@ double tentropd (double* x,double* y,int iLen,int nbins,int xpast,int ypast,int
   dsum=0.0;
   *sig=-1e6;//init sig to neg
   *nTE = 0.0;
+  /* 2nd cntpx should be cntpxy? */
```

Comment:
I added this (there's a C compiler warning) and was happy to leave it in, but I can remove it if you like.

Reply:
@olupton : the reason for `nitpicking` was following :

* often mod files or models are reused by other users
* these mocels are downloaded via modeldb as zip tarballs
* if some other domain scientist download the model and try to understand the comment, I think it will be bit difficult  to understand the context? For example, was this comment from original author? As a new user, what am I supposed to do here?

As a developer we do `git blame` and know what we are talking about. But I think this is not how scientists will approach. So may be minimal thing we could do is to add bit context / reference to these additional comments? e.g. something like

> /* O. Lupton: I think 2nd cntpx should be cntpxy. See  https://github.com/ModelDBRepository/136095/pull/1*/

Just as a suggestion !

(I know we are just picking here few from large pool 😅)

Reply:
In this case I just made what is very likely the correct fix...